### ISSUE TITLE
Bower "unable to find a suitable version of polymer" on empty clean Polymer project

### ISSUE THREAD

**Issue by davie-robertson** [PredixDev/px-tree#8]
I have create an empty polymer project using the polymer-cli and the first thing I wanted to do was try the px-tree web component, however this is the error I get when trying to install:

```
Unable to find a suitable version for polymer, please choose one by typing one of the numbers below:
    1) polymer#^1.9.0 which resolved to 1.11.0 and is required by px-app-nav#2.0.11, px-icon-set#2.0.17, px-tree#2.0.5
    2) polymer#^1.9.1 which resolved to 1.11.0 and is required by px-app-helpers#1.4.2, px-branding-bar#1.0.6
    3) polymer#1.9 - 2 which resolved to 2.2.0 and is required by app-layout#2.0.4, iron-a11y-keys-behavior#2.0.1, iron-behaviors#2.0.0, iron-collapse#2.0.0, iron-dropdown#2.0.0, iron-fit-behavior#2.0.1, iron-flex-layout#2.0.0, iron-icon#2.0.1, iron-iconset-svg#2.1.0, iron-media-query#2.0.0, iron-meta#2.0.2, iron-overlay-behavior#2.1.0, iron-resizable-behavior#2.0.1, iron-scroll-target-behavior#2.0.0, iron-selector#2.0.0, neon-animation#2.0.1, px-theme#3.1.2
    4) polymer#^2.0.0 which resolved to 2.2.0 and is required by my-app

Prefix the choice with ! to persist it to bower.json

? Answer 3

Unable to find a suitable version for webcomponentsjs, please choose one by typing one of the numbers below:
    1) webcomponentsjs#^0.7.24 which resolved to 0.7.24 and is required by polymer#1.11.0
    2) webcomponentsjs#^1.0.0 which resolved to 1.0.17 and is required by my-app
    3) webcomponentsjs#^v1.0.2 which resolved to 1.0.17 and is required by polymer#2.2.0

Prefix the choice with ! to persist it to bower.json

? Answer 3

```

edit: took a guess and added the next questions - now to see if it works.
Any suggestions?

**Comment by mdwragg** [comment]
Predix UI components are (for now) built against Polymer 1.x. The docs explain this: https://www.predix-ui.com/#/about/start-developing.

TL;DR - resolve for polymer 1.x and webcomponents 0.7.x
